tests/pixels/test_decoder_base.py

```diff
@@ -933,6 +935,46 @@ def test_validate_plugins(self):
         with pytest.raises(ValueError, match=msg):
             decoder._validate_plugins("foo")
 
+    @pytest.mark.skipif(not HAVE_NP, reason="NumPy is not available")
+    def test_bitpack_option_array(self):
+        """Test compatibility of "is_bitpacked" option with array methods."""
+        decoder = get_decoder(RLELossless)
+        source = RLE_1_1_3F.ds
+
+        decoder.as_array(source, is_bitpacked=False)
```

Comment:
Do you need to have `is_bitpacked` as an option for the `Decoder` methods?

Reply:
I'm not sure I understand this question. Is it related to [this](https://github.com/pydicom/pydicom/pull/2216#issuecomment-3123383220) or something else?

Reply:
With changes in [0248155](https://github.com/pydicom/pydicom/pull/2216/commits/024815540ede6bae53acc8a1daf09785cb582c91) is is no longer necessary to specify `is_bitpacked` like this, and this test has been removed anyway because the behaviour it tested is no longer required.